app/schema_validation/service_join_request.py

```diff
@@ -10,3 +10,39 @@
     "required": ["requester_id", "contacted_user_ids"],
     "additionalProperties": False,
 }
+
+service_join_request_update_schema = {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "description": "Schema for updating a service join request",
+    "type": "object",
+    "properties": {
+        "permissions": {
+            "type": "array",
+            "items": {
+                "type": "string",
+                "enum": [
+                    "manage_users",
+                    "manage_templates",
+                    "manage_settings",
+                    "send_texts",
+                    "send_emails",
+                    "send_letters",
+                    "manage_api_keys",
+                    "platform_admin",
```

Comment:
Optional, but we might want to remove `platform_admin` from here for safety, since it should never be possible to set that this way and we don't want users to ever have it.